app/js/renderer.js

```diff
@@ -83,6 +83,7 @@ function updateBody(data, settings, scripts) {
   }
   $('#content').html(generatedHtml);
 
+  refreshIframe();
```

Comment:
Ok for the plural.
The purpose is really close to the implementation but the goal is to force the execution of "dynamic tag" (iframe, script...) because we set the content of the page instead of letting the browser load the content. Does that make sense ?